api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -1830,6 +1834,9 @@ def transfer_with_liquid_class(
         :param group_wells: For multi-channel transfers only. If set to ``True``, group together contiguous wells
             given into a single transfer step, taking into account the tip configuration. If ``False``, target
             each well given with the primary nozzle. Defaults to ``True``.
+        :param keep_last_tip: If set to ``True``, do not drop or return the last tip used in the transfer. If set to
+            ``False``, the last tip will be dropped or returned. If not set, default depends on tip policy chosen. A
+            tip policy of ``"never"`` will default to ``True``, all other tip policies will default to ``False``.
```

Comment:
Prose suggestions below. Highest priority change is to use the term ``new_tip`` (the public parameter name) instead of "tip policy" (an internal implementation name).
```suggestion
        :param keep_last_tip: When ``True``, the pipette keeps the last tip used in the transfer attached. When
            ``False``, the last tip will be dropped or returned. If not set, behavior depends on the value of ``new_tip``.
            ``new_tip="never"`` keeps the tip, and all other values of ``new_tip`` drop or return the tip.
```

Reply:
And replicate any changes here in the other two functions, modulo action name (transfer/consolidate/distribute).